docs/src/develop/GetStarted.htm

```diff
@@ -49,6 +49,8 @@
             </li>
             <li>
             	<p>Use the Grommet generator to bootstrap your new app</p>
+              <p>If you get 'Could not find Grommit' error make sure you set the
```

Comment:
It doesn't show on my machine. I've checked with both VIM and atom. I have editorconfig plugin. Can't figure it out.